decidim-core/app/mailers/decidim/reported_mailer.rb

```diff
@@ -21,14 +21,27 @@ def report(user, report)
       end
     end
 
-    def hide(user, report)
+    def hidden_by_admin(user, report, current_user)
       with_user(user) do
         @report = report
         @participatory_space = @report.moderation.participatory_space
         @reportable = @report.moderation.reportable
         @organization = user.organization
         @user = user
-        subject = I18n.t("hide.subject", scope: "decidim.reported_mailer")
+        @admin = current_user
+        subject = I18n.t("hidden_by_admin.subject", scope: "decidim.reported_mailer", admin: @admin.name)
+        mail(to: user.email, subject:)
+      end
+    end
+
+    def hidden_automatically(user, report)
```

Comment:
```suggestion
    # This is meant to be used when a resource is hidden by an algorithm, such as the `decidim-ai` module, or the `Decidim.max_reports_before_hiding` feature.
    def hidden_automatically(user, report)
```

Reply:
Suggestion applied in 8555f7f9dd . Done without Github's help as we needed to rename a file.